common/dtd/ldmlSupplemental.dtd

```diff
@@ -1225,6 +1225,25 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 <!ATTLIST coverageLevel match CDATA #REQUIRED >
     <!--@MATCH:any-->
 
+<!ELEMENT pathGroups ( pathGroup+ ) >
+<!ELEMENT pathGroup ( pathMatch* ) >
+<!ATTLIST pathGroup id NMTOKEN #REQUIRED >
+    <!--@MATCH:any-->
+<!ATTLIST pathGroup includeAll NMTOKENS #IMPLIED wip -  >
+    <!--@MATCH:any-->
+
+<!-- a pathMatch is basically a coveragelevel minus thevalue -->
+<!ELEMENT pathMatch EMPTY >
+<!ATTLIST pathMatch inLanguage CDATA #IMPLIED >
+    <!--@MATCH:any-->
+<!ATTLIST pathMatch inScript CDATA #IMPLIED >
+    <!--@MATCH:validity/script-->
+<!ATTLIST pathMatch inTerritory CDATA #IMPLIED >
+    <!--@MATCH:any-->
```

Comment:
Uh, why not `@MATCH:validity/region`?

Reply:
```suggestion
    <!--@MATCH:validity/region-->
```

sure... maybe coverageLevel needs that also